### PR TITLE
Update 22.vaccines.md

### PR DESCRIPTION
**Pull request by nilswellhausen**
Finished up Novavax.
@rando2 @RLordan @ajlee21

### CODE REVIEW

File: content/22.vaccines.md [hunk 1]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+The outcomes of this trial suggest that Novavax-CoV2373 has an acceptable safety profile and is able to induce a strong immune response with high neutralizing antibody titers.
+The phase 2 component of this phase 1/2 trial was recently uploaded to an open-access repository [@doi:10.1101/2021.02.26.21252482].
+This part of the trial was designed to identify which dosing regimen should move forward into late phase clinical trials.
+Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5ug or 25ug Novavax-CoV2373 or placebo in two doses, 21 days apart.
+In line with the phase 1 data, reactogenicity remained mild to moderate and of short duration. 
+Both doses were able to induce high anti-spike IgG titers as well as neutralizing antibody responses after the second dose.
+Based on both safety and efficacy data, the 5 ug dosing regimen was selected as the optimal dose regiment for the ongoing phase 3 trial.
+Although the phase 3 trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase 3 trial in the UK and South Africa.
```
Comment (agitter) [inline content/22.vaccines.md:466]: Is there a press release or website we can cite for this?

File: content/22.vaccines.md [hunk 2]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+This trial included over 15.000 participants in the UK and 4000 participants in SA with occurrence of a PCR-confirmed symptomatic case as the primary endpoint.
```
Comment (agitter) [inline content/22.vaccines.md:467]:
```suggestion
This trial included over 15,000 participants in the UK and 4,000 participants in South Africa with occurrence of a PCR-confirmed symptomatic case as the primary endpoint.
```

**Review by agitter — COMMENTED** [review]
I did a light review, and it reads well to me.  To minor formatting suggestions are:
- I and II for trial phases
- `&mu;g` instead of `ug`

File: content/22.vaccines.md [hunk 3]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
 These findings were subsequently evaluated in a baboon primate model, in which Novavax-CoV2373 also elicited high antibody titers against the SARS-CoV-2 spike protein, as well as an antigen specific T-cell response.
 Based on this data Novavax initiated a Phase 1/2 clinical trial to evaluate the safety and immunogenicity of Novavax-CoV2373 with Matrix-M (NCT04368988)[@doi:10.1056/NEJMoa2026920].
 
+The phase 1/2 trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms.
```
Comment (rando2) [inline content/22.vaccines.md:451]: Consider adding citation here too or rearranging the paragraph break?

File: content/22.vaccines.md [hunk 4]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+Secondary outcomes measured included virus neutralization, T-cell responses, and unsolicited adverse events.
+The authors reported that no treatment-related adverse serious events occurred in any of the treatment arms.
+Reactogenicity was mostly absent and of short duration.
+The two-dose vaccine regimen induced anti-spike IgG levels and neutralizing antibody-titers exceeding those in convalescent plasma of symptomatic patients.
+In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response.
+The outcomes of this trial suggest that Novavax-CoV2373 has an acceptable safety profile and is able to induce a strong immune response with high neutralizing antibody titers.
+The phase 2 component of this phase 1/2 trial was recently uploaded to an open-access repository [@doi:10.1101/2021.02.26.21252482].
+This part of the trial was designed to identify which dosing regimen should move forward into late phase clinical trials.
+Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5ug or 25ug Novavax-CoV2373 or placebo in two doses, 21 days apart.
```
Comment (rando2) [inline content/22.vaccines.md:462]:
```suggestion
Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 &mu;g or 25 &mu;g Novavax-CoV2373 or placebo in two doses, 21 days apart.
```

File: content/22.vaccines.md [hunk 5]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+In line with the phase 1 data, reactogenicity remained mild to moderate and of short duration. 
+Both doses were able to induce high anti-spike IgG titers as well as neutralizing antibody responses after the second dose.
+Based on both safety and efficacy data, the 5 ug dosing regimen was selected as the optimal dose regiment for the ongoing phase 3 trial.
```
Comment (rando2) [inline content/22.vaccines.md:465]:
```suggestion
Based on both safety and efficacy data, the 5 &mu;g dosing regimen was selected as the optimal dose regiment for the ongoing phase 3 trial.
```

File: content/22.vaccines.md [hunk 6]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+Reactogenicity was mostly absent and of short duration.
+The two-dose vaccine regimen induced anti-spike IgG levels and neutralizing antibody-titers exceeding those in convalescent plasma of symptomatic patients.
+In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response.
+The outcomes of this trial suggest that Novavax-CoV2373 has an acceptable safety profile and is able to induce a strong immune response with high neutralizing antibody titers.
+The phase 2 component of this phase 1/2 trial was recently uploaded to an open-access repository [@doi:10.1101/2021.02.26.21252482].
+This part of the trial was designed to identify which dosing regimen should move forward into late phase clinical trials.
+Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5ug or 25ug Novavax-CoV2373 or placebo in two doses, 21 days apart.
+In line with the phase 1 data, reactogenicity remained mild to moderate and of short duration. 
+Both doses were able to induce high anti-spike IgG titers as well as neutralizing antibody responses after the second dose.
```
Comment (rando2) [inline content/22.vaccines.md:464]:
```suggestion
Both dose levels were able to induce high anti-spike IgG titers as well as neutralizing antibody responses after the second dose.
```

File: content/22.vaccines.md [hunk 7]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+Based on both safety and efficacy data, the 5 ug dosing regimen was selected as the optimal dose regiment for the ongoing phase 3 trial.
+Although the phase 3 trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase 3 trial in the UK and South Africa.
+This trial included over 15.000 participants in the UK and 4000 participants in SA with occurrence of a PCR-confirmed symptomatic case as the primary endpoint.
+In the first interim analysis (UK), 56 cases of COVID-19 were observed in the placebo group compared to 6 cases in the treatment group.
+Importantly, the vaccine candidate also shows significant clinical efficacy against the prevelant UK and South African variant.
```
Comment (rando2) [inline content/22.vaccines.md:469]:
```suggestion
Importantly, the vaccine candidate also shows significant clinical efficacy against the prevelant UK and South African variants.
```

File: content/22.vaccines.md [hunk 8]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+The company also has initiated the development of new constructs to select candidates that can be used as a booster against new strains and plans to initiate clinical trials for these new constructs in the second quarter of 2021.
```
Comment (rando2) [inline content/22.vaccines.md:470]:
```suggestion
The company has also initiated the development of new constructs to select candidates that can be used as a booster against new strains and plans to initiate clinical trials for these new constructs in the second quarter of 2021.
```

**Review by rando2 — COMMENTED** [review]
This is super interesting! I just did a light review without digging into the science for this first pass.

File: content/22.vaccines.md (hunk 3 above)
Comment (nilswellhausen) [reply content/22.vaccines.md:451]:
```suggestion
The phase 1/2 trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms [@doi:10.1056/NEJMoa2026920].
```

File: content/22.vaccines.md (hunk 1 above)
Comment (RLordan) [reply content/22.vaccines.md:466]: Same for the phase 1 data :)

File: content/22.vaccines.md [hunk 9]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
 These findings were subsequently evaluated in a baboon primate model, in which Novavax-CoV2373 also elicited high antibody titers against the SARS-CoV-2 spike protein, as well as an antigen specific T-cell response.
 Based on this data Novavax initiated a Phase 1/2 clinical trial to evaluate the safety and immunogenicity of Novavax-CoV2373 with Matrix-M (NCT04368988)[@doi:10.1056/NEJMoa2026920].
 
+The phase 1/2 trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms [@doi:10.1056/NEJMoa2026920].
+Participants that received the recombinant SARS-CoV-2 vaccine with or without Matrix-M adjuvant got two injections, 21 days apart.
```
Comment (RLordan) [inline content/22.vaccines.md:452]:
```suggestion
Participants that received the recombinant SARS-CoV-2 vaccine with or without the Matrix-M adjuvant got two injections, 21 days apart.
```

**Review by RLordan — APPROVED** [review]
A cool vaccine. Just the reference needed for the phase I study. Thanks @nilswellhausen

File: content/22.vaccines.md [hunk 10]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
 These findings were subsequently evaluated in a baboon primate model, in which Novavax-CoV2373 also elicited high antibody titers against the SARS-CoV-2 spike protein, as well as an antigen specific T-cell response.
 Based on this data Novavax initiated a Phase 1/2 clinical trial to evaluate the safety and immunogenicity of Novavax-CoV2373 with Matrix-M (NCT04368988)[@doi:10.1056/NEJMoa2026920].
 
+The phase 1/2 trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms [@doi:10.1056/NEJMoa2026920].
```
Comment (ajlee21) [inline content/22.vaccines.md:451]:
```suggestion
The phase I/II trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms [@doi:10.1056/NEJMoa2026920].
```

Comment (ajlee21) [reply content/22.vaccines.md:451]: Mainly to be consistent with the other text

File: content/22.vaccines.md [hunk 11]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+Participants that received the recombinant SARS-CoV-2 vaccine with or without Matrix-M adjuvant got two injections, 21 days apart.
+Primary outcomes that were assessed include reactogenicity, lab-values (serum chemistry and hematology), and anti-spike IgG levels.
+Secondary outcomes measured included virus neutralization, T-cell responses, and unsolicited adverse events.
+The authors reported that no serious treatment-related adverse events occurred in any of the treatment arms.
+Reactogenicity was mostly absent and of short duration.
+The two-dose vaccine regimen induced anti-spike IgG levels and neutralizing antibody-titers exceeding those in the convalescent plasma of symptomatic patients.
+In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response.
+The outcomes of this trial suggest that Novavax-CoV2373 has an acceptable safety profile and is able to induce a strong immune response with high neutralizing antibody titers.
+The phase 2 component of this phase 1/2 trial was recently uploaded to an open-access repository [@doi:10.1101/2021.02.26.21252482].
```
Comment (ajlee21) [inline content/22.vaccines.md:460]:
```suggestion
The phase II component of this phase I/II trial was recently uploaded to an open-access repository [@doi:10.1101/2021.02.26.21252482].
```

File: content/22.vaccines.md [hunk 12]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+This part of the trial was designed to identify which dosing regimen should move forward into late phase clinical trials.
+Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 &mu;g or 25 &mu;g Novavax-CoV2373 or placebo in two doses, 21 days apart.
```
Comment (ajlee21) [inline content/22.vaccines.md:462]: Is the placebo just the adjuvant I assume...just glancing through the paper I think thats the case, but just wanted to confirm with you

File: content/22.vaccines.md [hunk 13]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+In line with the phase 1 data, reactogenicity remained mild to moderate and of short duration.
```
Comment (ajlee21) [inline content/22.vaccines.md:463]:
```suggestion
In line with the phase I data, reactogenicity remained mild to moderate and of short duration. 
```

File: content/22.vaccines.md [hunk 14]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response.
+The outcomes of this trial suggest that Novavax-CoV2373 has an acceptable safety profile and is able to induce a strong immune response with high neutralizing antibody titers.
+The phase 2 component of this phase 1/2 trial was recently uploaded to an open-access repository [@doi:10.1101/2021.02.26.21252482].
+This part of the trial was designed to identify which dosing regimen should move forward into late phase clinical trials.
+Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 &mu;g or 25 &mu;g Novavax-CoV2373 or placebo in two doses, 21 days apart.
+In line with the phase 1 data, reactogenicity remained mild to moderate and of short duration. 
+Both dose levels were able to induce high anti-spike IgG titers as well as neutralizing antibody responses after the second dose.
+Based on both safety and efficacy data, the 5 &mu;g dosing regimen was selected as the optimal dose regiment for the ongoing phase 3 trial.
```
Comment (ajlee21) [inline content/22.vaccines.md:465]:
```suggestion
Based on both safety and efficacy data, the 5 &mu;g dosing regimen was selected as the optimal dose regiment for the ongoing phase III trial.
```

File: content/22.vaccines.md [hunk 15]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+Although the phase 3 trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase 3 trial in the UK and South Africa.
```
Comment (ajlee21) [inline content/22.vaccines.md:466]:
```suggestion
Although the phase III trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase 3 trial in the UK and South Africa.
```

Comment (ajlee21) [inline content/22.vaccines.md:466]:
```suggestion
Although the phase III trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase III trial in the UK and South Africa (SA).
```

**Review by ajlee21 — APPROVED** [review]
Really interesting read! Had very minor comments

File: content/22.vaccines.md (hunk 1 above)
Comment (nilswellhausen) [reply content/22.vaccines.md:466]: Yes: https://ir.novavax.com/news-releases/news-release-details/novavax-covid-19-vaccine-demonstrates-893-efficacy-uk-phase-3

File: content/22.vaccines.md (hunk 12 above)
Comment (rando2) [reply content/22.vaccines.md:462]: Hmmm I'm not sure they explain what it was! I might leave a comment to ask the authors. There was another clinical trial of the adjuvant alone but it's not clear what they did here.

For now, I'm making a note here so we can merge this in.
```suggestion
Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 &mu;g or 25 &mu;g Novavax-CoV2373 or placebo in two doses, 21 days apart.<!--To Do: What is placebo?-->
```

Comment (nilswellhausen) [reply content/22.vaccines.md]: I think adjuvant only was done previously already. The company uses this adjuvant for other vaccines as well so they did not have to test the adjuvant on its own anymore.

File: content/22.vaccines.md [hunk 16]
```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
...
+Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 &mu;g or 25 &mu;g Novavax-CoV2373 or placebo in two doses, 21 days apart.<!--To Do: What is placebo?-->
+In line with the phase I data, reactogenicity remained mild to moderate and of short duration. 
+Both dose levels were able to induce high anti-spike IgG titers as well as neutralizing antibody responses after the second dose.
+Based on both safety and efficacy data, the 5 &mu;g dosing regimen was selected as the optimal dose regiment for the ongoing phase III trial.
+Although the phase III trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase 3 trial in the UK and South Africa.
+This trial included over 15,000 participants in the UK and 4,000 participants in South Africa with occurrence of a PCR-confirmed symptomatic case as the primary endpoint.
+In the first interim analysis (UK), 56 cases of COVID-19 were observed in the placebo group compared to 6 cases in the treatment group.
+Importantly, the vaccine candidate also shows significant clinical efficacy against the prevelant UK and South African variants.
```
Comment (rando2) [inline content/22.vaccines.md:469]:
```suggestion
Importantly, the vaccine candidate also shows significant clinical efficacy against the prevalent UK and South African variants.
```

**Pull request merged**